### PR TITLE
Code Review Preparation; Micro-Reddit

### PR DESCRIPTION
**Pull request by jamezjaz**
**This is the Micro-Reddit project built with Ruby on Rails framework.**
On this branch, we did the following:
- We generated the user, post, and comment models.
- We created the models' associations.
- We validated the various columns.

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :posts
```
Comment (mosesogwo) [inline app/models/comment.rb]: A comment belongs to a post, **not posts**

File: README.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code and inspiration was used
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
+
+
 # README
```
Comment (mosesogwo) [inline README.md]: This section should be removed

File: README.md [hunk 3]
```diff
@@ -1,3 +1,44 @@
+# Micro-Reddit
+This is a project built based on the Active Record (Model) principle. It is built with Ruby on Rails frame work.
```
Comment (mosesogwo) [inline README.md:2]: OPTIONAL: You could add a brief description of the project features here - the models and associations you have created.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello team:clap: Great job! Please see my comments on your comment.rb and README files**

_**Kindly make those changes and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great work implementing the given suggestions.

**You may proceed to the next project.**

**Good luck :four_leaf_clover: 
Simon Wathigo**

**Pull request merged**